test/scripts/run_test.py

```diff
@@ -47,47 +48,51 @@ def analyse_instrumentation_test_result(result):
     return ok_num
 
 
-def analyse_unit_test_result(result):
+def analyse_unit_test_result(result, mode, module):
+    xml_file_name = module + '-' + mode.value + '-result-' + XML_SUFFIX
     total_num = 0
-    fail_num = 0
+    success_num = 0
+    p = re.compile(".*?XML test result file generated at (.*?)\. Total tests (\d+).*?passed (\d+)")
```

Comment:
This is already a structural xml file, why are you still using regular express on it?

Reply:
then the whole PR is unnecessary .........

Reply:
done